docs/reference/troubleshooting/common-issues/hotspotting.asciidoc

```diff
@@ -263,7 +263,7 @@ further insight on it via <<tasks,the task management API>>,
 
 [source,console]
 ----
-GET _tasks?human&detailed
+GET _tasks?human&detailed&pretty
```

Comment:
No need for this change I think, folks will either be running this in the Kibana dev console (which prettifies the output itself) or else they have some other way to prettify the output.

```suggestion
GET _tasks?human&detailed
```